core/src/main/java/org/infinispan/configuration/parsing/Attribute.java

```diff
@@ -214,7 +214,8 @@
     ZERO_CAPACITY_NODE,
     INVALIDATION_THRESHOLD,
     CONTEXT_INITIALIZERS,
-    GROUPER;
+    GROUPER,
+    DEFAULT_CLUSTER;
```

Comment:
we could reuse the `cluster` attribute IMO. 
Like we do for the endpoints: 
`<endpoints security-realm="abc">` configures the default but it can be overwritten in the child `<endpoint security-reaml="xyz">`

Reply:
Ok